tket/src/Predicates/PassGenerators.cpp

```diff
@@ -592,7 +596,7 @@ PassPtr DecomposeTK2(const Transforms::TwoQbFidelities& fid) {
 }
 
 PassPtr ThreeQubitSquash(bool allow_swaps) {
-  Transform t = Transforms::two_qubit_squash() >>
+  Transform t = Transforms::two_qubit_squash(allow_swaps) >>
```

Comment:
Does this change mean that the maximum number of CX gates in a 2-qubit circuit is less than 20 now, or are there still cases when it is 20?

Reply:
No. `two_qubit_squash` will still require up to three CX in the worst case.